packages/variants/src/variants.ts

```diff
@@ -148,7 +174,24 @@ export function getSoftVariant(theme: IPartialTheme): ITheme {
     // inputBackgroundCheckedHovered: p.themeDarkAlt,
     inputForegroundChecked: p.themeLighter,
     // inputFocusBorderAlt: p.themePrimary,
-    variantBorder: !fullTheme.isInverted ? p.neutralLight : p.neutralLighterAlt
+    variantBorder: !fullTheme.isInverted ? p.neutralLight : p.neutralLighterAlt,
+
+    buttonBackground: !fullTheme.isInverted ? p.themeLighterAlt : p.themeLight,
+    buttonText: !fullTheme.isInverted ? p.neutralPrimary : p.themePrimary,
+    buttonBorder: p.neutralSecondary,
+    buttonBackgroundHovered: !fullTheme.isInverted ? p.themeLighter : changeOpacity(p.themeTertiary, 50),
```

Comment:
so this can just be:

updateA(getColorFromString(p.themeTertiary), 50).str

Reply:
gah okay, it's because getColorFromString() might return undefined
in that case keep your wrapper and have it do basic error checking:

let c = getColorFromString(inputColor);
return c ? updateA(c, 50).str : inputColor;